io.httpie.HTTPie.yml

```diff
@@ -0,0 +1,85 @@
+app-id: io.httpie.HTTPie
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+separate-locales: false
+tags:
+  - proprietary
+command: httpie-desktop
+rename-icon: httpie
+rename-desktop-file: httpie.desktop
+finish-args:
+  - --socket=x11
+  - --share=network
+  - --share=ipc
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+  - name: httpie
+    buildsystem: simple
+    build-commands:
+      - unappimage httpie.AppImage
+      - rm httpie.AppImage
+
+      - install -Dm755 httpie.sh /app/bin/httpie-desktop
+      - install -dm755 /app/share/applications
+      - desktop-file-edit --set-key=Exec --set-value="httpie-desktop %U" squashfs-root/httpie.desktop
+      - install -Dm644 squashfs-root/httpie.desktop -t /app/share/applications
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/512x512/apps/httpie.png
+        -t /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/256x256/apps/httpie.png
+        -t /app/share/icons/hicolor/256x256/apps
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/128x128/apps/httpie.png
+        -t /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/64x64/apps/httpie.png
+        -t /app/share/icons/hicolor/64x64/apps
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/32x32/apps/httpie.png
+        -t /app/share/icons/hicolor/32x32/apps
+      - install -Dm644 squashfs-root/usr/share/icons/hicolor/16x16/apps/httpie.png
+        -t /app/share/icons/hicolor/16x16/apps
+
+      - install -Dm644 io.httpie.HTTPie.appdata.xml -t /app/share/metainfo/
+      - mv squashfs-root /app/main/
+    sources:
+      - type: file
```

Comment:
This needs to use extra-data as it's proprietary.

Reply:
@barthalion ,
I have asked upstream whether they permit Flathub to redistribute their Application

Reply:
https://github.com/httpie/desktop/issues/26#issuecomment-1465788625